### PR TITLE
Added AuthorisationPermissionMismatch to the exclusion criteria on St…

### PR DESCRIPTION
**Pull request by emt2600**
Added AuthorisationPermissionMismatch to the exclusion criteria on Storage Account Analysis to prevent failure when a databricks storage account is present

### CODE REVIEW

File: pwsh/dev/functions/processStorageAccountAnalysis.ps1 [hunk 1]
```diff
@@ -35,7 +35,7 @@ function processStorageAccountAnalysis {
 
                 $urlServiceProps = "$($storageAccount.Properties.primaryEndpoints.blob)?restype=service&comp=properties"
                 $saProperties = AzAPICall -AzAPICallConfiguration $azAPICallConf -uri $urlServiceProps -method 'GET' -listenOn 'Content' -currentTask "$($storageAccount.name) get restype=service&comp=properties" -saResourceGroupName $resourceGroupName
-                if ($saProperties -eq 'AuthorizationFailure' -or $saProperties -eq 'AuthorizationPermissionDenied' -or $saProperties -eq 'ResourceUnavailable') {
+                if ($saProperties -eq 'AuthorizationFailure' -or $saProperties -eq 'AuthorizationPermissionDenied' -or $saProperties -eq 'ResourceUnavailable' or $saProperties -eq 'AuthorizationPermissionMismatch' ) {
```
Comment (JulianHayward) [inline pwsh/dev/functions/processStorageAccountAnalysis.ps1:38]: -or

**Review by JulianHayward — CHANGES_REQUESTED** [review]
-or

File: pwsh/dev/functions/processStorageAccountAnalysis.ps1 (hunk 1 above)
Comment (emt2600) [reply pwsh/dev/functions/processStorageAccountAnalysis.ps1]: updated

**Comment by MarcelHeek** [comment]
Not sure wheter this is in the same script context, but sometimes I get fatal error on this in the Storage Analysis section with `Unexpected Error`

..........
careblob get restype=service&comp=properties #1 'Unexpected Error' occurred (trying 10 times); sleep 2 seconds
Connection timed out
careblob get restype=service&comp=properties #2 'Unexpected Error' occurred (trying 10 times); sleep 3 seconds
Connection timed out
 careblob get restype=service&comp=properties #3 'Unexpected Error' occurred (trying 10 times); sleep 5 seconds
Connection timed out
  Forced DEBUG: careblob get restype=service&comp=properties -> attempt#4 processing: careblob get restype=service&comp=properties uri: 'https://careblob.blob.core.windows.net/?restype=service&comp=properties'
  Forced DEBUG: careblob get restype=service&comp=properties -> unexpectedError: true
 careblob get restype=service&comp=properties #4 'Unexpected Error' occurred (trying 10 times); sleep 7 seconds
......... 
Connection timed out
  Forced DEBUG: careblob get restype=service&comp=properties -> attempt#11 processing: careblob get restype=service&comp=properties uri: 'https://careblob.blob.core.windows.net/?restype=service&comp=properties'
  Forced DEBUG: careblob get restype=service&comp=properties -> unexpectedError: true
 careblob get restype=service&comp=properties #11 'Unexpected Error' occurred (tried 5 times)/exit
Exception: /home/runner/work/azure-governance-visualizer/azure-governance-visualizer/tools/pwsh/AzGovVizParallel.ps1:11310
 Line |
11310 |          $storageAccounts | ForEach-Object -Parallel ***
      |          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      | Error - check the last console output for details

**Pull request merged**